frontend-react/src/utils/misc.ts

```diff
@@ -46,6 +46,38 @@ export const checkJson = (
     }
 };
 
+export function includesSpecialChars(text: string): boolean {
+    return /[^a-z\d\s]/gi.test(text);
+}
+
+export const isProhibitedOrgName = (
+    orgName: string
+): { prohibited: boolean; errorMsg: string } => {
+    const prohibitedNames = [
+        "sender",
+        "receiver",
+        "org",
+        "organization",
+        "list",
+        "all", //api/check issue
+        "revs", //org/{organizationName}/senders/revs issue
+    ];
+
+    if (prohibitedNames.indexOf(orgName) > -1) {
```

Comment:
I _think_ this is a case sensitive check. So would we want to allow `All`? (I'm not sure, but probably not.)

Maybe the orgName should be lowerCased at the start of this function so it doesn't depend on it happening higher up?